src/bp-activity/bp-activity-functions.php

```diff
@@ -6043,3 +6072,47 @@ function bb_is_group_activity_comment( $comment = 0 ) {
 
 	return false;
 }
+
+/**
+ * Function to create a paginated backgroud job for activity following notifications.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param array $args  Array of arguments.
+ * @param array $paged Current page number for pagination.
+ */
+function bb_create_background_activity_following_post_notification( $parse_args, $paged = 1 ) {
```

Comment:
```suggestion
function bb_create_background_activity_following_post_notification( $args, $paged = 1 ) {
```

And update where its used into the function.